tokio-util/tests/framed_write.rs

```diff
@@ -155,13 +155,13 @@ impl AsyncWrite for Mock {
         _cx: &mut Context<'_>,
         buf: &[u8],
     ) -> Poll<Result<usize, io::Error>> {
-        match Pin::get_mut(self).write(buf) {
+        match std::io::Write::write(&mut Pin::get_mut(self), buf) {
```

Comment:
import `use std::io::Write` above this line?

Reply:
@LucioFranco OK, will try.

Reply:
I could actually remove the `std::io::` qualifier, it's no longer ambiguous for some reason (used to be).